core/src/components/checkbox/checkbox.scss

```diff
@@ -148,6 +148,53 @@ input {
   opacity: 0;
 }
 
+// Checkbox Bottom Content
+// ----------------------------------------------------------------
+
+.checkbox-bottom {
+  @include padding(5px, null, null, null);
+
+  display: flex;
+
+  justify-content: space-between;
+
+  font-size: dynamic-font(12px);
+
+  white-space: normal;
+}
+
+:host(.checkbox-label-placement-stacked) .checkbox-bottom {
+  font-size: dynamic-font(16px);
```

Comment:
I added this because without it the text is really small.

Reply:
Same as above* :)